integration

```diff
@@ -69,6 +69,7 @@
   "Aohzan/hass-prixcarburant",
   "Aohzan/ipx800",
   "Aohzan/ipx800v5",
+  "Arbuzov/home_assistant_delonghi_primadonna",
```

Comment:
This is added as an integration, but hacs.json points to a card (plugin) <https://github.com/Arbuzov/home_assistant_delonghi_primadonna/blob/1.5.3/hacs.json>